### PR TITLE
Missing translations

### PR DESCRIPTION
**Pull request by CamilleLegeron**
Make translatable more strings

### CODE REVIEW

**Review by paulfitz — COMMENTED** [review]
Thanks @CamilleLegeron for going through all that! Great to have more elements translatable.

Along with the specific comments I had, I've a couple of observations that don't need addressing in the PR but would be good to think about for future polishing:
  * There are places in the code where words are CAPITALIZED for styling purposes. It seems excessive to have translators making multiple versions of translations for that. Could be worth moving the capitalization to css (e.g. text-transform: capitalize). @dsagal what do you think?
  * Paul Janzen, who volunteered translations for Portuguese and German, found some words/phrases hard to translate out of context. The pattern of having a sentence/paragraph with some internal variable piece of it translated separately makes sense in code, but the context gets lost. I wonder if we could avoid this fragmentation at least for minor formatting like italics. What do you think @berhalak?

File: static/locales/en.client.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+    "ACLUsers": {
```
[the remark this answers is not in the document]
Comment (CamilleLegeron) [reply static/locales/en.client.json]: It was indeed my fault, I've done what you propose and I think it's fine now

File: static/locales/de.client.json [hunk 2]
```diff
@@ -664,7 +664,7 @@
         "Current field ": "Aktuelles Feld ",
         "OK": "OK"
     },
-    "TypeTransformation": {
+    "TypeTransform": {
```
[the remark this answers is not in the document]
Comment (CamilleLegeron) [reply static/locales/de.client.json:669]: I just pull the main branch, I hope that there will be no rapid change

**Review by paulfitz — APPROVED** [review]
Thanks @CamilleLegeron !

**Pull request merged**